README.md

```diff
@@ -186,9 +187,37 @@ fmt.Println(formattedStr)
 //   main.readFile | .../example/main.go | 6
 ```
 
-### Writing your own custom format
+### Writing custom output format
 
-`eris` also allows advanced users to write their own custom formats in case the default format doesn't fit your requirements. The [`UnpackedError`](https://godoc.org/github.com/rotisserie/eris#UnpackedError) object provides a convenient and developer friendly way to store and access existing error traces. The `ErrRoot` and `ErrChain` fields correspond to the root error and wrap error chain, respectively. If any other error type is unpacked, it will appear in the `ExternalErr` field. You can access all of the information contained in an error via [`eris.Unpack`](https://godoc.org/github.com/rotisserie/eris#Unpack).
+`eris` also allows advanced users to construct custom error strings or objects in case the default error doesn't fit the requirements. The [`UnpackedError`](https://godoc.org/github.com/rotisserie/eris#UnpackedError) object provides a convenient and developer friendly way to store and access existing error traces. The `ErrRoot` and `ErrChain` fields correspond to the root error and wrap error chain, respectively. If any other error type is unpacked, it will appear in the `ExternalErr` field. You can access all of the information contained in an error via [`eris.Unpack`](https://godoc.org/github.com/rotisserie/eris#Unpack).
```

Comment:
I think `the requirements` makes less sense than `your requirements`. Maybe change it to `their requirements`?